administrator/components/com_content/src/Model/ArticlesModel.php

```diff
@@ -305,9 +311,16 @@ protected function getListQuery()
         }
 
         // Filter by featured.
-        $featured = (string) $this->getState('filter.featured');
+        $featured = $this->isFeatured();
```

Comment:
We need to keep the filter because it is used, for instance, the quickicon.
```suggestion
        $featured = (string) $this->getState('filter.featured', '');
        if ($featured === '') {
            $featured = $this->isFeatured();
        }
```